data/icons/Makefile.am

```diff
@@ -5,11 +5,12 @@ public_icons_themes = \
 	$(NULL)
 
 public_icons = \
-	hicolor_apps_16x16_caja.png \
-	hicolor_apps_22x22_caja.png \
-	hicolor_apps_24x24_caja.png \
-	hicolor_apps_32x32_caja.png \
-	hicolor_apps_scalable_caja.svg \
+	#TODO create these files
+	#hicolor_apps_16x16_caja.png \
+	#hicolor_apps_22x22_caja.png \
+	#hicolor_apps_24x24_caja.png \
+	#hicolor_apps_32x32_caja.png \
+	#hicolor_apps_scalable_caja.svg \
```

Comment:
why you comment these lines instead delete them?

Reply:
I'm not a programmer, actually i always do this because i usually mess up XD.... This pull request is supper trivial if you noticed.